_posts/2020-09-15-newpipe-0.20.0.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "NewPipe 0.20.0 released: A new major version, a new unified player, a new notification experience... A lot is new this time!"
+short: "NewPipe 0.20.0 released"
+date: 2020-09-15 05:00:00 +0100
+categories: [pinned, announcement]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Hey there! Guess who is back? We are. Admittedly, we're quite a bit late with this release. But there is a _very_ good reason for that. Are you ready for it? Sit down, grab some popcorn and a beverage of your choice. This is going to be long.
+<!-- more -->
+
+## A new unified player
+
+"What in the name of the coronavirus is the _unified player_?" you ask. Well, first of all, that's a very weird expression. Where did you even get that one from... - you know what? Save it. We don't want to know. You just keep on using that, we'll keep on talking about the new player.
```

Comment:
Why `...` instead of `…`?

Reply:
uhm, to show more hesitation I guess?

Reply:
How do you even type the ellipsis in a simple text editor? In something like Office, three periods get replaced by it.

Reply:
With most GUI toolkits on Linux, it's Ctrl+Shift+U 2026